include/openenclave/host.h

```diff
@@ -68,6 +68,12 @@ OE_EXTERNC_BEGIN
 /**
  * @cond DEV
  */
+
+/**
+ *  Flag passed into oe_create_enclave to run the enclave with kss mode.
```

Comment:
Flag reserved for internal use to indicate the enclave was configured to run in kss mode